test/server/core/gameSteps/regroupWindow/RegroupPhase.spec.ts

```diff
@@ -119,9 +111,16 @@ describe('Regroup phase', function() {
 
                     const { context } = contextRef;
 
+                    // Play card Attack pattern delta
                     context.player1.clickCard(context.attackPatternDelta);
+                    // Select Ardent Sympathizer to get +3/+3
                     context.player1.clickCard(context.ardentSympathizer);
+                    // Select Scout Bike Pursuer to get +2/+2
                     context.player1.clickCard(context.scoutBikePursuer);
+                    // Select General Krell to receive +1/+1
+                    context.player1.clickCard(context.generalKrell);
+
+                    // Select wampa to attack Ardent Sympathizer
```

Comment:
nit: maybe note that this attack damages it enough to be defeated when the effect expires so the intent is clear

Reply:
gotcha!